test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
The fact that `indexOf` returns a number is not a problem. If the return value is greater than or equal to zero, the list/string contains the value/substring.